src/wasm-builder.h

```diff
@@ -677,6 +677,8 @@ class Builder {
         value = Literal(bytes.data());
         break;
       }
+      case anyref:
+        assert(false && "anyref not implemented yet");
       case exnref:
         // TODO Implement and return nullref
```

Comment:
We can put this TODO above the both `anyref` and `exnref` cases because it applies to both of them